src/recordlinker/schemas/pii.py

```diff
@@ -323,11 +324,17 @@ def feature_iter(self, feature: Feature) -> typing.Iterator[str]:
                 if address.postal_code:
                     # only use the first 5 digits for comparison
                     yield address.postal_code[:5]
-        elif feature == Feature.FIRST_NAME:
+        elif feature == Feature.GIVEN_NAME:
             for name in self.name:
                 for given in name.given:
                     if given:
                         yield given
+        elif feature == Feature.FIRST_NAME:
+            for name in self.name:
+                # We only want the first given name for comparison
+                for given in name.given[0:1]:
```

Comment:
Are we safe to assume that the first name of a patient will always be the first index of the `name.given` array? I guess it doesn't matter because users could choose to check the full `name.given` array by using the `GIVEN_NAME` feature if they really wanted to.

Reply:
I think that is a fine assumption, as long the collection remains a list as those or orderable (opposed to something like a set, which is not).  Are you thinking that this could be an issue in the FHIR payloads we parse?

Reply:
Okay cool, the fact that the list is orderable makes sense to me as I was thinking which spot in the `given` array would the most recent first name be.  And no, I think it should be fine with FHIR since the given is always a list